platforms/nuttx/src/px4/stm/stm32_common/version/board_mcu_version.c

```diff
@@ -151,8 +153,10 @@ int board_mcu_version(char *rev, const char **revstr, const char **errata)
 		chip_errata = NULL;
 		break;
 
-	case MCU_REV_STM32F7_REV_V:
-		*rev = 'V';
+	case MCU_REV_STM32F4_REV_5:
+		// MCU_REV_STM32F4_REV_B shares the same REV_ID
+		// MCU_REV_STM32F7_REV_V shares the same REV_ID
+		*rev = chip_version == STM32H74xx_75xx ? 'V' : '5';
```

Comment:
The comment does not match the code
STM32**H**74xx_75xx

Reply:
Yes, sorry, I got confused by surrounding code.

Judging by the ST errata sheets, there's no V or X revisions for STM32F7 series, only STM32H7. I've changed the enum names to reflect that.

Reply:
comment says MCU_REV_STM32**F**7_REV_V shares the same REV_ID code has  STM32**H**74xx_75xx

Reply:
Yes, I've changed the comment to say H7.

The comment was about the enum name of `MCU_REV_STM32F7_REV_V`, which is itself somewhat confusing, since no F7 to date seems to have a `REV_ID` of `0x2003`.